configs/config_sepolia.py

```diff
@@ -55,3 +55,143 @@
 
 WSTETH_TOKEN = "0xB82381A3fBD3FaFA77B3a7bE693342618240067b"
 CHAIN_DEPOSIT_CONTRACT = "0x7f02C3E3c98b133055B8B348B2Ac625669Ed295D"
+
+EASYTRACK_SIMPLE_DVT_TRUSTED_CALLER = ""
+EASYTRACK_SIMPLE_DVT_ADD_NODE_OPERATORS_FACTORY = ""
+EASYTRACK_SIMPLE_DVT_SET_VETTED_VALIDATORS_LIMITS_FACTORY = ""
+
+ACCOUNTING_ORACLE_IMPL = None
```

Comment:
Do we really needed in empty variables here?

Reply:
Yes, we do. To be able the python code be compilable.